ConnectorAutocadCivil/ConnectorAutocadCivil/UI/ConnectorBindingsAutocadCivil.Receive.cs

```diff
@@ -223,29 +219,32 @@ string id
           // convert base (or base fallback values) and store in appobj converted prop
           if (commitObj.Convertible)
           {
-            converter.Report.Log(commitObj); // Log object so converter can access
-            try
+            if (StoredObjects.TryGetValue(commitObj.OriginalId, out Base obj))
             {
-              commitObj.Converted = ConvertObject(commitObj, converter);
+              converter.Report.Log(commitObj); // Log object so converter can access
+              commitObj.Converted = ConvertObject(obj, converter);
```

Comment:
We're sure this call won't throw?

Reply:
yes it returns an empty list if the conversion throws in the converter, it was only caught before because the converter method wasn't wrapped with a trycatch